test_log_json test fails for the borg binaries

```
gw2] darwin -- Python 3.4.0 /vagrant/borg/borg/.tox/py34/bin/python3.4
self = <borg.testsuite.archiver.ArchiverTestCaseBinary testMethod=test_log_json>

    def test_log_json(self):
        self.create_test_files()
        self.cmd('init', '--encryption=repokey', self.repository_location)
        log = self.cmd('create', '--log-json', self.repository_location + '::test', 'input', '--list', '--debug')
        messages = {}  # type -> message, one of each kind
        for line in log.splitlines():
            msg = json.loads(line)
            messages[msg['type']] = msg
    
        file_status = messages['file_status']
        assert 'status' in file_status
        assert file_status['path'].startswith('input')
    
        log_message = messages['log_message']
        assert isinstance(log_message['time'], float)
        assert log_message['levelname'] == 'DEBUG'  # there should only be DEBUG messages
>       assert log_message['name'].startswith('borg.')
E       AssertionError: assert False
E        +  where False = <built-in method startswith of str object at 0x109b6b078>('borg.')
E        +    where <built-in method startswith of str object at 0x109b6b078> = 'pyimod03_importers'.startswith

.tox/py34/lib/python3.4/site-packages/borg/testsuite/archiver.py:1680: AssertionError
```

--- a/src/borg/testsuite/archiver.py
+++ b/src/borg/testsuite/archiver.py
@@ -1678,7 +1678,6 @@ class ArchiverTestCase(ArchiverTestCaseBase):
         log_message = messages['log_message']
         assert isinstance(log_message['time'], float)
         assert log_message['levelname'] == 'DEBUG'  # there should only be DEBUG messages
-        assert log_message['name'].startswith('borg.')
         assert isinstance(log_message['message'], str)
 
     def test_common_options(self):

